Woof/Woof/View/SitterFlow/EditSitterInformationView.swift

```diff
@@ -0,0 +1,106 @@
+import SwiftUI
+
+/// A view to edit the sitter information.
+struct EditSitterInformationView: View {
+    /// The name of the sitter.
+    @Binding var name: String
+
+    /// The surname of the sitter.
+    @Binding var surname: String
+
+    /// The phone of the sitter.
+    @Binding var phone: String
+
+    /// The additional information about the sitter, like his experience with dogs, favourite places for walks,
+    /// special skills, certificates, etc.
+    @Binding var bio: String
+
+    /// The price per hour for walking charged by the sitter.
+    @Binding var pricePerHour: String
+
+    var body: some View {
+        VStack {
+            Text("Edit your information")
+            TextField("Enter your name", text: $name)
+            TextField("Enter your surname", text: $surname)
+            TextField("Enter your phone", text: $phone)
+            TextField("Enter your price per hour for walking", text: $pricePerHour)
+                .keyboardType(.decimalPad)
```

Comment:
Should we use the `keyboardType` modifier for other Views (like `.phonePad` for the phone's textfield)?